client/app/cloud/project/compute/infrastructure/privateNetwork/cloud-project-compute-infrastructure-privateNetwork-list.directive.js

```diff
@@ -321,11 +339,12 @@ class PrivateNetworkListCtrl {
     onKeyDown ($event) {
         switch ($event.which) {
             case 27:
-                //Important not to put $event.preventDefault(); before the switch statement since it will catch and prevent default
-                //behavior on keyDown everywhere in the directive, inputs included.
+                // Important not to put $event.preventDefault(); before the switch statement since it will catch and prevent default
+                // behavior on keyDown everywhere in the directive, inputs included.
                 $event.preventDefault();
                 this.hideDialog();
                 break;
+            default:
```

Comment:
Needed by ESlint

Reply:
We have to re-enable ESLint. For quite a long time the `grunt` task has been commented (ref: https://github.com/ovh-ux/ovh-manager-cloud/blob/master/Gruntfile.js#L960)